altair/utils/theme.py

```diff
@@ -16,12 +17,12 @@
     from altair.utils.plugin_registry import PluginEnabler
     from altair.vegalite.v5.theme import AltairThemes, VegaThemes
 
-ThemeType = Plugin[Dict[str, Any]]
+ThemeType = Plugin[ThemeConfig]
```

Comment:
Keeping this `ThemeType` alias for backwards compat, but using `ThemeConfig` in annotations.

`ThemeConfig` has full docs and can be used to navgiate through the other `TypedDict`(s)